tests/helpers.py

```diff
@@ -369,10 +380,8 @@ def delete_all_storageclass():
         kind=constants.STORAGECLASS,
         namespace=defaults.ROOK_CLUSTER_NAMESPACE
     )
-    storageclass_list = get_all_storageclass_name()
-    for item in storageclass_list:
-        logger.info(f"Deleting StorageClass with name {item}")
-        assert SC.delete(resource_name=item)
+    logger.info(f"Deleting StorageClass with name {sc_name}")
+    assert SC.delete(resource_name=sc_name)
```

Comment:
return SC.delete(resource_name=sc_name)

and delete the next line - return True